xml/security_ldap.xml

```diff
@@ -889,7 +663,258 @@ telephoneNumber: +49 1234 567-10</screen>
 Linux,ou=devel,dc=example,dc=com</screen>
   </sect2>
  </sect1>
- <sect1 xml:id="sec.security.ldap.info">
+
+<sect1 xml:id="sec.security.ldap.yast.usergr">
+  <title>Configuring LDAP Users and Groups in &yast;</title>
+<!--taroth 2019-02-25: let's see where to move/integrate the content from this
+    section in the end-->
+  <para>
+   The actual registration of user and group data differs only slightly from
+   the procedure when not using LDAP. The following instructions relate to
+   the administration of users. The procedure for administering groups is
+   analogous.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Access the &yast; user administration with <menuchoice>
+     <guimenu>Security and Users</guimenu><guimenu>User and Group
+     Management</guimenu></menuchoice>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Use <guimenu>Set Filter</guimenu> to limit the view of users to the
+     LDAP users and enter the password for Root DN.
```

Comment:
There should never be a need to enter the root DN password, outside of an administration context. Perhaps this means the bind-dn and bind-password for pam/nsswitch?

Reply:
Maybe I'm misunderstanding, is this yast changing LDAP server data? Or configuring a server to consume LDAP data?

Reply:
AFAIK, you could configure LDAP users on the OpenLDAP server with YaST, yes. YaST is (open)SUSE's graphical tool for system administration tasks. So individual YaST modules are designed to access and alter configuration files, if required, otherwise they could not fulfill this task.

Reply:
AHhhhh I see. Okay this makes more sense now if it's yast changing the users. In this case,  Ithink my comments don't mean much :)